graylog2-web-interface/src/views/components/searchbar/saved-search/SearchActionsMenu.tsx

```diff
@@ -160,6 +163,16 @@ const SearchActionsMenu = () => {
     loadAsDashboard(history, view);
   }, [history, view]);
 
+  const isDirty = useIsDirty();
+
+  useHotkeys('SAVE', () => {
+    saveSearch(title);
+  }, { scopes: 'search', enabled: (isDirty && loaded) }, [loaded, isDirty, saveSearch, title]);
```

Comment:
IMO we should not disable the save button.
- when there are no changes, we can still save the search and display the user notification, similar to e.g. a Google Drive document.
- when the search has not been saved yet, we can open the dialog to save it (similar to "save as")

In every case we should avoid displaying the browser dialog, to save the website as HTML when clicking this shortcut.